src/software/utils/main_fisheyeProjection.cpp

```diff
@@ -0,0 +1,431 @@
+// This file is part of the AliceVision project.
+// Copyright (c) 2017 AliceVision contributors.
+// Copyright (c) 2016 openMVG contributors.
+// This Source Code Form is subject to the terms of the Mozilla Public License,
+// v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include <aliceVision/numeric/numeric.hpp>
+#include <aliceVision/image/io.hpp>
+#include <aliceVision/image/Sampler.hpp>
+#include <aliceVision/image/convertion.hpp>
+#include <aliceVision/system/Logger.hpp>
+#include <aliceVision/system/cmdline.hpp>
+#include <aliceVision/camera/camera.hpp>
+
+#include <dependencies/vectorGraphics/svgDrawer.hpp>
+
+#include <boost/filesystem.hpp>
+#include <boost/program_options.hpp> 
+
+#include <OpenImageIO/imageio.h>
+#include <OpenImageIO/imagebuf.h>
+#include <OpenImageIO/imagebufalgo.h>
+
+#include <string>
+#include <iostream>
+#include <iterator>
+#include <fstream>
+#include <vector>
+#include <boost/regex.hpp>
+
+// These constants define the current software version.
+// They must be updated when the command line is changed.
+#define ALICEVISION_SOFTWARE_VERSION_MAJOR 1
+#define ALICEVISION_SOFTWARE_VERSION_MINOR 0
+
+using namespace aliceVision;
+
+namespace fs = boost::filesystem;
+namespace po = boost::program_options;
+namespace oiio = OIIO;
+
+
+/**
+ * @brief Function to map 3D coordinates onto a 2D image according a spherical projection
+ */
+class SphericalMapping
```

Comment:
can't those two static methods be just normal functions? Is the class necessary?